tslint.json

```diff
@@ -90,7 +90,8 @@
     "validate-decorators": [true, {
       "Component": {
         "encapsulation": "\\.None$",
-        "moduleId": "^module\\.id$"
+        "moduleId": "^module\\.id$",
+        "preserveWhitespaces": "false$"
```

Comment:
super nit: is there any reason not to make it `"^false$"`?

Reply:
Nah, but it's really the same either way